setup/03_optional-docker-environment/README.md

```diff
@@ -27,55 +27,60 @@ git clone https://github.com/rasbt/LLMs-from-scratch.git
 cd LLMs-from-scratch
 ```
 
+2. Move the `.devcontainer` folder from `setup/03_optional-docker-environment/` to the current directory (project root).
+
 ```bash
 mv setup/03_optional-docker-environment/.devcontainer ./
 ```
 
-3. In Docker Desktop, make sure that ***desktop-linux* builder** is running and will be used to build the Docker container (see *Docker Desktop* -> *Change settings* -> *Builders* -> *desktop-linux* -> *...* -> *Use*)
+3. In Docker Desktop, make sure that **_desktop-linux_ builder** is running and will be used to build the Docker container (see _Docker Desktop_ -> _Change settings_ -> _Builders_ -> _desktop-linux_ -> _..._ -> _Use_)
```

Comment:
I've been using Markdown for 10+ years and didn't know `_` was the the same as `*` :D

Reply:
Yeah, I only use `_` as an alternative to `*` to improve markdown readability (I use a VS Code extension doing this markdown readability improvement job me). BTW, same goes for two asterisks/underscores for bold print like `**text**` and `__text__`. The asterisk syntax is just more robust and less prone to rendering issues.